doc/Attestation.adoc

```diff
@@ -7,32 +7,31 @@ A high level description of the thinking and how this can be used can be found
 at https://developers.yubico.com/PIV/Introduction/PIV_attestation.html
 
 === Usage
-Attestation works through a special key slot called “f9” this comes
-pre-loaded from factory with a key and cert signed by Yubico, but can be
-overwritten.
-After a key has been generated in a normal slot it can be attested by this
-special key, this can be realised by using the yubico-piv-tool action attest:
+The only purposes of PIV attestation is to show that a certain asymmetric key has been generated on device and not imported.
+
+Attestation works through a special key slot called “F9” that comes
+pre-loaded from the factory with a key and certificate signed by Yubico's root PIV CA. While not recommended for most use cases, this certificate can be overwritten. Note that, if overwritten, the original signing key and certificate in slot F9 cannot be restored. The attestation key and certificate will not be cleared out by a reset of the device.
```

Comment:
I'd put the advanced overwriting f9 use case in a separate document, a footnote, or at least an "advanced" section at the bottom.